### ISSUE TITLE
Variables in urls

### ISSUE THREAD

**Issue by davidgilbertson** [sasstools/sass-lint#63]
I have this SCSS

``` css
.value-proposition {
    $imagesUrl: '/images/';
    background: center url($imagesUrl + 'slide-01-bg.png') no-repeat;
    background-size: cover;
}
```

Which generates this error:

```
c:\www\mysite\node_modules\sass-lint\lib\groot.js:21
    throw new Error('Parsing error' + fileInfo + ': ' + e.message);
          ^
Error: Parsing error at app/components/App/migrated-sass/_CBE_project_defaults.scss: Please check validity of the block
starting from line #5
    at module.exports (c:\www\mysite\node_modules\sass-lint\lib\groot.js:21:11)
    at Function.sassLint.lintText (c:\www\mysite\node_modules\sass-lint\index.js:23:13)
    at c:\www\mysite\node_modules\sass-lint\index.js:73:21
    at Array.forEach (native)
    at Function.sassLint.lintFiles (c:\www\mysite\node_modules\sass-lint\index.js:72:9)
    at detectPattern (c:\www\mysite\node_modules\sass-lint\bin\sass-lint.js:16:18)
    at Object.<anonymous> (c:\www\mysite\node_modules\sass-lint\bin\sass-lint.js:54:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

Let me know if you want a separate issue, but the line number is also mis-reported. On a large file... ha, I just worked it out while typing this, the reported line number is exactly double the correct number minus one.

**Comment by Snugug** [comment]
This is, again, an issue with our AST. If you could report the issue [on the Gonzales issue queue](https://github.com/tonyganch/gonzales-pe/issues) and link this issue so we can track it, that'd be awesome.

**Comment by davidgilbertson** [comment]
It looks like the `gonzales-pe` issue [Concatenation does not work within `url` in SCSS](https://github.com/tonyganch/gonzales-pe/issues/85).

**Comment by davidgilbertson** [comment]
For reference, using sass string interpolation works
```css
.value-proposition {
    $imagesUrl: '/images/';
    background: center url('#{$imagesUrl}slide-01-bg.png') no-repeat;
    background-size: cover;
}
```
With or without the quote marks.

**Comment by DanPurdy** [comment]
This is still an issue as of version 3.2.1 of gonzales-pe, I have updated the original issue filed with the gonzales-pe repo as linked to above.

**Comment by bgriffith** [comment]
Update - This is still broken in `gonzales-pe@3.2.6`.

**Comment by bgriffith** [comment]
This is fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!
